app/main.py

```diff
@@ -1,8 +1,33 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is a class variable intended to store all `Person` instances by their names. Ensure that this dictionary is correctly used and updated throughout the code.